File: flutter/test_app/lib/data/supabase/model/DTOs/sportas_dto.dart
class SportasDto {
  final String idKorisnika;
  final String datumRodenja;
  final String? tipClanarine;

  SportasDto({
    required this.idKorisnika,
    required this.datumRodenja,
    this.tipClanarine,
  });

  factory SportasDto.fromJson(Map<String, dynamic> json) {
    return SportasDto(
      idKorisnika: json['IdKorisnika'] as String,
      datumRodenja: json['DatumRodenja'] as String,
      tipClanarine: json['TipClanarine'] as String?,
    );
  }
}

File: kotlin_jetpack_compose/app/src/main/java/com/example/testapp/data/supabase/model/DTOs/SportasDTO.kt
package com.example.testapp.data.supabase.model.DTOs

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class SportasDTO(
    @SerialName("IdKorisnika")
    val idKorisnika: String,
    @SerialName("DatumRodenja")
    val datumRodenja: String,
    @SerialName("TipClanarine")
    val tipClanarine: String? = null
)